npm/asciidoctor-revealjs

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+
+'use strict'
+
+process.title = 'asciidoctor-revealjs'
+
+const pkg = require('../package.json')
+const { Options, Invoker, processor } = require('@asciidoctor/cli')
+const asciidoctor = require('@asciidoctor/core')();
+const asciidoctorRevealjs = require('@asciidoctor/reveal.js')
+const path = require('path')
+const EventEmitter = require('events')
```

Comment:
You don't need this part (it's an implementation detail of a Mutex). In this case, we don't need a Mutex.